src/NuGet.Clients/NuGet.CommandLine/Common/Console.cs

```diff
@@ -282,12 +297,55 @@ public void PrintJustified(int startIndex, string text, int maxWidth)
                     // Print it with the correct padding
                     Out.WriteLine((leftPadding > 0) ? content.PadLeft(leftPadding) : content);
 
+                    if (content.Length == text.Length)
+                    {
+                        break;
+                    }
+
                     // Get the next substring to be printed
-                    text = text.Substring(content.Length);
+                    text = text.Substring(newLineIndex == -1 ? content.Length : crlf ? content.Length + 2 : content.Length + 1);
                 }
             }
         }
 
+
```

Comment:
Add a block-comment here explaining why this method is necessary and what it does over Environment.NewLine.

Reply:
I've done this in the latest commit. Let me know if I can improve the explanation provided by the comment.